Hometask_4_2/Hometask_4_2.py

```diff
@@ -0,0 +1,51 @@
+import random
+import string
+
+
+# Create the list dictionaries from 2 to 10.
+def create_dicts():
+    output = []  # Dicts list
+    for _ in range(random.randint(2, 10)):  # Dictionary counter
+        dictionary = {}
+        for _ in range(random.randint(1, 26)):  # Elements counter
+            # Key(random letter), value(random number from 0 to 100)
+            dictionary[random.choice(string.ascii_lowercase)] = random.randint(0, 100)
+        output.append(dictionary)  # Append every dictionary to the list
+    return output
+
+
+all_values = create_dicts()
+
+
+def all_values_dict():
```

Comment:
Please, rework names of ALL functions.
E.g. "create_dict_with_all_values" or "create_preprocessed_dict".

It's a good practice to start your function names with verbs like "get_...", "delete_...", "process_..." and so on.